build: Make `$(package)_*_env` available to all `$(package)_*_cmds`

hebasto opened the pull request:
On master (1e7564eca8a688f39c75540877ec3bdfdde766b1) the depends build system, which is based on pure GNU Make, works, but it lacks robustness, and in some corner cases it fails. For example, see bitcoin/bitcoin#22552.

Another [bug](https://github.com/bitcoin/bitcoin/issues/22719) in the depends build system has already become a problem at least two times in the past (https://github.com/bitcoin/bitcoin/pull/16883#issuecomment-683817472 and https://github.com/bitcoin/bitcoin/pull/24134). Each time the problem was solved with other means.

The initial [solution](https://github.com/bitcoin/bitcoin/pull/19882) had some discussion. Also it was discussed on the IRC meeting in #bitcoin-core-builds channel. This PR, actually, is a resurrection of it, as the bug silently struck pretty [recently](https://github.com/bitcoin/bitcoin/pull/24134).

The bug is well described in bitcoin/bitcoin#22719.

Here is another, a bit simpler description, which requires only basic shell (bash, dash etc) experience.
After creating targets by this code:https://github.com/bitcoin/bitcoin/blob/1e7564eca8a688f39c75540877ec3bdfdde766b1/depends/funcs.mk#L280 a "draft" line of recipe like `$($(1)_config_env) $(call $(1)_config_cmds, $(1))` becomes a shell command sequence `VAR1=foo VAR2=bar command1 && command2` which is supposed to be executed in a [new sub-shell](https://www.gnu.org/software/make/manual/html_node/Execution.html#Execution).

Please note that `VAR1=foo VAR2=bar` part is visible for the first `command1` only (for details see shell docs). Example:
```sh
$ VAR1="foo" VAR2="bar" echo "begin" && printenv VAR1 && printenv VAR2 && echo "end"
begin
$ echo $?
1
```

Using the `export` command is a trivial solution:
```sh
$ export VAR1="foo" VAR2="bar"; echo "begin" && printenv VAR1 && printenv VAR2 && echo "end"
begin
foo
bar
end
$ echo $?
0
```

As a [new sub-shell](https://www.gnu.org/software/make/manual/html_node/Execution.html#Execution) is invoked for each line of the recipe, there are no side effects of using `export`. It means this solution should not be considered invasive.

Fixes bitcoin/bitcoin#22719.

---

Also this PR removes no longer needed crutch from `qt.mk`.

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
b4a3dc838a410d03e27937c55a724f2dc8b35b038953c997a100cf422d3103fe  guix-build-c86da12d2007/output/aarch64-linux-gnu/SHA256SUMS.part
5c7f55f22f04f0337932eb5e6b2f491418a566e83d8606016a99d173c6de2e9f  guix-build-c86da12d2007/output/aarch64-linux-gnu/bitcoin-c86da12d2007-aarch64-linux-gnu-debug.tar.gz
3d98de08458043397be488c631cb01dda05d1af81de5acdf22ac113c20150425  guix-build-c86da12d2007/output/aarch64-linux-gnu/bitcoin-c86da12d2007-aarch64-linux-gnu.tar.gz
37715e6a5464ed1ba3e841c1a917d34abf64215d854822f8b819323d1e1125e0  guix-build-c86da12d2007/output/arm-linux-gnueabihf/SHA256SUMS.part
07827a0f905113f258a0843fe0dafd6f1575d449230e7469eb48d1ea29a6137f  guix-build-c86da12d2007/output/arm-linux-gnueabihf/bitcoin-c86da12d2007-arm-linux-gnueabihf-debug.tar.gz
844b8214b2eea7aaf7b346e7330dda9b259de6ded720672f8ab80dc191937393  guix-build-c86da12d2007/output/arm-linux-gnueabihf/bitcoin-c86da12d2007-arm-linux-gnueabihf.tar.gz
feddb82595ed497a6e1859fb9a26f1853c3630e3eb19bd667e6adc3b31e1d155  guix-build-c86da12d2007/output/arm64-apple-darwin/SHA256SUMS.part
03c0300e868b5d8b01a00db389fe89c1811da4d170988dd4bd6d047cb2f65c1c  guix-build-c86da12d2007/output/arm64-apple-darwin/bitcoin-c86da12d2007-arm64-apple-darwin.tar.gz
b3c893b12eafc8ccb7cb2b7d0f03397f21bbbdf090289333c241be36a1223f0a  guix-build-c86da12d2007/output/arm64-apple-darwin/bitcoin-c86da12d2007-osx-unsigned.dmg
fed2590dda6f73483ecf205d3fd650831e896813e0801bff8ddc549cd8ae9b95  guix-build-c86da12d2007/output/arm64-apple-darwin/bitcoin-c86da12d2007-osx-unsigned.tar.gz
53b64ec1edae9a65cc6884958bb4657106d366523c807994aa1c48d1c96c6c5f  guix-build-c86da12d2007/output/dist-archive/bitcoin-c86da12d2007.tar.gz
7fb6c235b3708f6b2739085ff4a2257ad0ef17b7ce63a9c7e38f1a8ecea45c66  guix-build-c86da12d2007/output/powerpc64-linux-gnu/SHA256SUMS.part
3eeaa800ab8784c236b1a310c22fc74e8b145bda0505e36a59af12eff6214076  guix-build-c86da12d2007/output/powerpc64-linux-gnu/bitcoin-c86da12d2007-powerpc64-linux-gnu-debug.tar.gz
e51881b2db486da9695626c185d6809122a7b95bb5ccfe4419882734c696ed89  guix-build-c86da12d2007/output/powerpc64-linux-gnu/bitcoin-c86da12d2007-powerpc64-linux-gnu.tar.gz
25b3c16762576f29ab8fcc92e61894b3fb3fe93472b313b523ad759ad64491f1  guix-build-c86da12d2007/output/powerpc64le-linux-gnu/SHA256SUMS.part
8dd5b59837acede88bb552fa50072d5bc0e1b1366e714065f954042c85079a92  guix-build-c86da12d2007/output/powerpc64le-linux-gnu/bitcoin-c86da12d2007-powerpc64le-linux-gnu-debug.tar.gz
2b778eeeadddb23aa6d50363554f5158c60ade26a5dd273f13e78de3f514f4d5  guix-build-c86da12d2007/output/powerpc64le-linux-gnu/bitcoin-c86da12d2007-powerpc64le-linux-gnu.tar.gz
3421f7de070c8749eb838724678e3c03340d4828d2ee7bae6e5c740326c7c94d  guix-build-c86da12d2007/output/riscv64-linux-gnu/SHA256SUMS.part
b402a1074372b575fe14816d48c1c7f6446e679a470e6abdfd4f75fc8fef565d  guix-build-c86da12d2007/output/riscv64-linux-gnu/bitcoin-c86da12d2007-riscv64-linux-gnu-debug.tar.gz
afff68a7d4d475064d0f6b43c551ffc0ce39038b9e8fd5ad573c26be336d153e  guix-build-c86da12d2007/output/riscv64-linux-gnu/bitcoin-c86da12d2007-riscv64-linux-gnu.tar.gz
9628956e57aea2f90ebe0752d1c7ff32ea9533262777f2aca0df215ee23cf120  guix-build-c86da12d2007/output/x86_64-apple-darwin/SHA256SUMS.part
eebc4489b3882f06de10e84eecbdc4266a3e85e155e6a9a7644bec2bf608a760  guix-build-c86da12d2007/output/x86_64-apple-darwin/bitcoin-c86da12d2007-osx-unsigned.dmg
630708f2d6fe0e20334a3a33c40b1386f3dde2c1deac58385248f6b00f84baf3  guix-build-c86da12d2007/output/x86_64-apple-darwin/bitcoin-c86da12d2007-osx-unsigned.tar.gz
914fcc739cc257c459f31164b2635fb9e1c7b36b3cbe71802a27428b2c3bf59d  guix-build-c86da12d2007/output/x86_64-apple-darwin/bitcoin-c86da12d2007-osx64.tar.gz
1ed1591b079e882c9e498d49aceeb3794b136a9ece222dc615f5270ad3cf83d4  guix-build-c86da12d2007/output/x86_64-linux-gnu/SHA256SUMS.part
3125fb9bec7a5234a204cfab5c85d593586412a176d5daf07926a8e6664e28db  guix-build-c86da12d2007/output/x86_64-linux-gnu/bitcoin-c86da12d2007-x86_64-linux-gnu-debug.tar.gz
cca2c5ae6e23aff59e02b7528d6405b0e10955c59a0ecf44e7a0f286bf621a46  guix-build-c86da12d2007/output/x86_64-linux-gnu/bitcoin-c86da12d2007-x86_64-linux-gnu.tar.gz
```

hebasto commented:
Rebased c86da12d20073bcc92fab064fdeb3d0f8ba988ae -> 626b79b777bb72f2bef611b383a289be69f62fd4 ([pr24279.01](https://github.com/hebasto/bitcoin/commits/pr24279.01) -> [pr24279.02](https://github.com/hebasto/bitcoin/commits/pr24279.02)) on top of the merged #24348.

hebasto commented:
#### Guix builds:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
95911d45edb992d70c3678c4fefb61c5c292b2cc8b99985f6e0c19067f720214  guix-build-626b79b777bb/output/aarch64-linux-gnu/SHA256SUMS.part
6d0705454e6c76fa21e958c37b8227fcc05ccee0990f54212b0271ca326a83f9  guix-build-626b79b777bb/output/aarch64-linux-gnu/bitcoin-626b79b777bb-aarch64-linux-gnu-debug.tar.gz
675ecb02efc5b12f1a087d861aa52f3b2d6c463da0dd97ec1698b4aca4c16171  guix-build-626b79b777bb/output/aarch64-linux-gnu/bitcoin-626b79b777bb-aarch64-linux-gnu.tar.gz
a35853a1b9c809897b88d1ea662ed807af662e7cdd6ab90a7781c47a66717502  guix-build-626b79b777bb/output/arm-linux-gnueabihf/SHA256SUMS.part
18ffdf2d6d4d6eae3fa0ee7953207cd406a275ae14ae569d27add62cbd4206bb  guix-build-626b79b777bb/output/arm-linux-gnueabihf/bitcoin-626b79b777bb-arm-linux-gnueabihf-debug.tar.gz
5a7e4c5c1692e22d608a66397dafce49e9a279177b4ebe882a95c0d2cc8d9a1a  guix-build-626b79b777bb/output/arm-linux-gnueabihf/bitcoin-626b79b777bb-arm-linux-gnueabihf.tar.gz
9d51371c8b613beb0946341b8cd19c6d1373bd012693c58a1896c0fedaf80610  guix-build-626b79b777bb/output/arm64-apple-darwin/SHA256SUMS.part
f121cbb558ce1f0fa92cefb87b723aaae5f756d5e27cf161041d3ad8f4908680  guix-build-626b79b777bb/output/arm64-apple-darwin/bitcoin-626b79b777bb-arm64-apple-darwin.tar.gz
4a00d0baff9dee997b3fd405fdc32873cce06e690dc2e9d8899d74d7620f1fe1  guix-build-626b79b777bb/output/arm64-apple-darwin/bitcoin-626b79b777bb-osx-unsigned.dmg
2c85f5f9114d93fa2a42f8d481aeb122c6b26986e1d2c9265bf32100e497abf7  guix-build-626b79b777bb/output/arm64-apple-darwin/bitcoin-626b79b777bb-osx-unsigned.tar.gz
8898cc98024dea6c3e8ddd2dc9768530c64fb0e2a6317f294f1731556bc3a71d  guix-build-626b79b777bb/output/dist-archive/bitcoin-626b79b777bb.tar.gz
a0e193ef29a337a6266bd808f23832b75dda68b0ce57d690f80c5cea945de4b6  guix-build-626b79b777bb/output/powerpc64-linux-gnu/SHA256SUMS.part
4f64070888c7b171642aa0d1b18ef8016750194b4ed53483b4e2fe3c3cddf022  guix-build-626b79b777bb/output/powerpc64-linux-gnu/bitcoin-626b79b777bb-powerpc64-linux-gnu-debug.tar.gz
07e0b8784b50b92137d3d910d499845b0996d2750e1dbeabdcf0cae6ec3f31b4  guix-build-626b79b777bb/output/powerpc64-linux-gnu/bitcoin-626b79b777bb-powerpc64-linux-gnu.tar.gz
1bc484289c28403b55d0996d1cb74b28ff0172a363fcfed51559deec6166f1c4  guix-build-626b79b777bb/output/powerpc64le-linux-gnu/SHA256SUMS.part
ef7f2a7d42f307e908bc4a508d9263b87145c712ba807c923e0a74d226070148  guix-build-626b79b777bb/output/powerpc64le-linux-gnu/bitcoin-626b79b777bb-powerpc64le-linux-gnu-debug.tar.gz
eccd0be92c2189ee98cb8ac632699e0da6b4e79a770823b502c66072e1ef80a5  guix-build-626b79b777bb/output/powerpc64le-linux-gnu/bitcoin-626b79b777bb-powerpc64le-linux-gnu.tar.gz
a093bed13d5f4482e56e26ca8d9f16632480275f9b107d499130dd2e9ba724bb  guix-build-626b79b777bb/output/riscv64-linux-gnu/SHA256SUMS.part
8a118ea9581b68e18087e169bf29c35dffec2a90c8dae943641bab060658fb92  guix-build-626b79b777bb/output/riscv64-linux-gnu/bitcoin-626b79b777bb-riscv64-linux-gnu-debug.tar.gz
550588d17491a826ff1f6741675779db6e4b42e8cbc1d9341302942c510ae0de  guix-build-626b79b777bb/output/riscv64-linux-gnu/bitcoin-626b79b777bb-riscv64-linux-gnu.tar.gz
dad901d4b9f17f0b576cac13fc9ac3d3d32b7339f1a4662a97fcd9fec7de5688  guix-build-626b79b777bb/output/x86_64-apple-darwin/SHA256SUMS.part
e1178983c78dd9630e4dee4341b25b67e80c26a6365fad7ee528f8d3fe497a62  guix-build-626b79b777bb/output/x86_64-apple-darwin/bitcoin-626b79b777bb-osx-unsigned.dmg
e1efa45cee23cb00aa4428aa6441c59044a101cc7db9a80f6a8dc001b13fa800  guix-build-626b79b777bb/output/x86_64-apple-darwin/bitcoin-626b79b777bb-osx-unsigned.tar.gz
c62865fd5162b0e41b465b214fefe10ecc5a8b521e0488323577d9f0e54958b2  guix-build-626b79b777bb/output/x86_64-apple-darwin/bitcoin-626b79b777bb-osx64.tar.gz
bf65014152b8d675003335a0145072837ad106c023ef2922473de5cff6a7e29e  guix-build-626b79b777bb/output/x86_64-linux-gnu/SHA256SUMS.part
ddbca5cc76b77af8dfe020b28f971d09f862008e7b03b613db319166d16a905d  guix-build-626b79b777bb/output/x86_64-linux-gnu/bitcoin-626b79b777bb-x86_64-linux-gnu-debug.tar.gz
78e574be4f93aa958493e01eddd074c02927d55886a8931665d8f951cd54b29b  guix-build-626b79b777bb/output/x86_64-linux-gnu/bitcoin-626b79b777bb-x86_64-linux-gnu.tar.gz
5463705429d8a215b4882158cb8c4920bcfc6e3f517cc80428dfb37d95d300c9  guix-build-626b79b777bb/output/x86_64-w64-mingw32/SHA256SUMS.part
71d6723c01113729f71c7ab0db8b5da8c43f152a07b345aeb1b0c15458924613  guix-build-626b79b777bb/output/x86_64-w64-mingw32/bitcoin-626b79b777bb-win-unsigned.tar.gz
9ccaaa63c14563fe24f79d71f4a5d92b037de044741308b4a0c64f2b13492008  guix-build-626b79b777bb/output/x86_64-w64-mingw32/bitcoin-626b79b777bb-win64-debug.zip
39d62e05d67a3221384d06b1c7f5c64f9f5d4d203d8af3e4979e834f0f0d8af9  guix-build-626b79b777bb/output/x86_64-w64-mingw32/bitcoin-626b79b777bb-win64-setup-unsigned.exe
1473e48573695762e2339d0390ce6593d48afae65845ec9affa9f637941afb66  guix-build-626b79b777bb/output/x86_64-w64-mingw32/bitcoin-626b79b777bb-win64.zip
```

hebasto commented:
Updated 626b79b777bb72f2bef611b383a289be69f62fd4 -> c8d338d0cad7e177397825621e6ddf3ebbe26e7a ([pr24279.02](https://github.com/hebasto/bitcoin/commits/pr24279.02) -> [pr24279.03](https://github.com/hebasto/bitcoin/commits/pr24279.03)):
- rebased due to the conflict with #19952
- removed no longer needed crutch from `qt.mk`

hebasto commented:
#### Guix builds on `x86_64`:
```
$ find guix-build-$(git rev-parse --short=12 HEAD)/output/ -type f -print0 | env LC_ALL=C sort -z | xargs -r0 sha256sum
f1a3777c3e7d7c8d59b16ac6d6085f501fe90e9b206366af49f3fafff2cf9a84  guix-build-3f90ddea8a6a/output/aarch64-linux-gnu/SHA256SUMS.part
7715f12e27852f30ee41f4b340c0e9ff44961ee831ed4c0eca25d5242b3a0fb4  guix-build-3f90ddea8a6a/output/aarch64-linux-gnu/bitcoin-3f90ddea8a6a-aarch64-linux-gnu-debug.tar.gz
6a2104f6014484cf7ea635539789748e729d64927380f206289480789b5bdcbc  guix-build-3f90ddea8a6a/output/aarch64-linux-gnu/bitcoin-3f90ddea8a6a-aarch64-linux-gnu.tar.gz
0742e51bfdc91f8521c51f88d8dd33cf1359e0f989cedcfa83e42a086989cab2  guix-build-3f90ddea8a6a/output/arm-linux-gnueabihf/SHA256SUMS.part
f6520f99a71bd81099f7ca057b7afedda8c04f8667655ccf85261a71d5b50098  guix-build-3f90ddea8a6a/output/arm-linux-gnueabihf/bitcoin-3f90ddea8a6a-arm-linux-gnueabihf-debug.tar.gz
e1f059327ff8c0d37517f147bdb3658269b32a589a6269b9714a86b597eaf901  guix-build-3f90ddea8a6a/output/arm-linux-gnueabihf/bitcoin-3f90ddea8a6a-arm-linux-gnueabihf.tar.gz
691e71ee82756c0da6ce088845d7381f754f54e643c5a3dc5e651dfde1cd4ffa  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/SHA256SUMS.part
b461e0e99feda343ae5c4d38294ba0d6436271c1d987db2c9fe7ee522989a946  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/bitcoin-3f90ddea8a6a-arm64-apple-darwin-unsigned.dmg
6194c67f4969a20e8ed8a270e6b518daa5bf7117bf8a0dc52f5bcc3b1d17e4d5  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/bitcoin-3f90ddea8a6a-arm64-apple-darwin-unsigned.tar.gz
813d71f1a7a9305879b05287c3d1688befdf1cacd3c75efe0fe8356dc0d84ddb  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/bitcoin-3f90ddea8a6a-arm64-apple-darwin.tar.gz
17125568ee8d8430064b9d907dbd8b00c27b2ec51279312d88a5485ca4cc50e5  guix-build-3f90ddea8a6a/output/dist-archive/bitcoin-3f90ddea8a6a.tar.gz
0b312c9b82982fb28ac0c4b0a2956ca41fec4dcfab920b63b7e5253735452120  guix-build-3f90ddea8a6a/output/powerpc64-linux-gnu/SHA256SUMS.part
efa52ae550949b9b335a64811f38e031fbc2669714401ed26b995ffd604602be  guix-build-3f90ddea8a6a/output/powerpc64-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64-linux-gnu-debug.tar.gz
54739498f4bf1f3b733e597c57334e984fe8033929390eb4704a18ef0f152ff3  guix-build-3f90ddea8a6a/output/powerpc64-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64-linux-gnu.tar.gz
621f0e34fdca59c9446b0768fc0ff8175ed9179d3a5f85bedc4492f72adddeed  guix-build-3f90ddea8a6a/output/powerpc64le-linux-gnu/SHA256SUMS.part
0f374f8c5d6796b93aef42f0da5e8cee6fa44fe71198c1c0eb90163da7d7f024  guix-build-3f90ddea8a6a/output/powerpc64le-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64le-linux-gnu-debug.tar.gz
cc6881853aec9c39945194b439fe28ab327bcbf65907fedb9861166f4101001a  guix-build-3f90ddea8a6a/output/powerpc64le-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64le-linux-gnu.tar.gz
93bb658a02282960f8110d0f14abac379693c8830cefa2de72e0d4ff1bc35d5c  guix-build-3f90ddea8a6a/output/riscv64-linux-gnu/SHA256SUMS.part
8dd848e5e57dd42e5f9960efd8e9e3b12540bfcd6bbd358462dea4fac5af2a8d  guix-build-3f90ddea8a6a/output/riscv64-linux-gnu/bitcoin-3f90ddea8a6a-riscv64-linux-gnu-debug.tar.gz
0180d91f039bd27ed201ca86b939f70bb0eabd023ffa264ada4276682c308dd3  guix-build-3f90ddea8a6a/output/riscv64-linux-gnu/bitcoin-3f90ddea8a6a-riscv64-linux-gnu.tar.gz
fba6ac013dc8a60752f418825fc6d45745918045c85408fb171b8c3dca13c86a  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/SHA256SUMS.part
9fd387d226cb0d4b5d7cfaae3b7de67e67d737026146d4f63234abd242e05655  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/bitcoin-3f90ddea8a6a-x86_64-apple-darwin-unsigned.dmg
8c1e6444a7e696d5ee96c6c4349d541c935660206c0cb4beb67a1393e22e9dbc  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/bitcoin-3f90ddea8a6a-x86_64-apple-darwin-unsigned.tar.gz
b579bbff39bfa905be1e6c6cecf1df40ddbb2c18337ff569183f5a3cec15d17f  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/bitcoin-3f90ddea8a6a-x86_64-apple-darwin.tar.gz
4085d46573c68d99a91341b0887e4e9613e39d6a2d9889f029adbf92dc87900c  guix-build-3f90ddea8a6a/output/x86_64-linux-gnu/SHA256SUMS.part
ac8683a938f33ab14da2ff62cb96e5655817aa755ef6d649db2b320a1d7b0f43  guix-build-3f90ddea8a6a/output/x86_64-linux-gnu/bitcoin-3f90ddea8a6a-x86_64-linux-gnu-debug.tar.gz
43ca0d57ae2a29a2b7ba4985f3bfa48a0acb9c9cf75e90bff7f4042d6990bfac  guix-build-3f90ddea8a6a/output/x86_64-linux-gnu/bitcoin-3f90ddea8a6a-x86_64-linux-gnu.tar.gz
60ed831c18881a3f552ab7a8bfffddd3f6e6c3095fbd17e95995f2819a885876  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/SHA256SUMS.part
59b19c36e73f7e00a507307580242752685f4e17fa21e3f6851b82c1fc095aea  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64-debug.zip
2ef548bee7f9f7420fa5336c3d830c6d40147d5082346ab8b594d38ef0ede74a  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64-setup-unsigned.exe
c4bc61a85609e838bc82c0fd9b73c43dbcc73261cb43bffda7ac6c2681de0eb0  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64-unsigned.tar.gz
0ae766295c82818aba22e2a5ed139a739fd1cfb5d0293d808408d9b46432c62d  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64.zip
```

jarolrod commented:
Guix hashes on x86, mine match @hebasto:

```
f1a3777c3e7d7c8d59b16ac6d6085f501fe90e9b206366af49f3fafff2cf9a84  guix-build-3f90ddea8a6a/output/aarch64-linux-gnu/SHA256SUMS.part
7715f12e27852f30ee41f4b340c0e9ff44961ee831ed4c0eca25d5242b3a0fb4  guix-build-3f90ddea8a6a/output/aarch64-linux-gnu/bitcoin-3f90ddea8a6a-aarch64-linux-gnu-debug.tar.gz
6a2104f6014484cf7ea635539789748e729d64927380f206289480789b5bdcbc  guix-build-3f90ddea8a6a/output/aarch64-linux-gnu/bitcoin-3f90ddea8a6a-aarch64-linux-gnu.tar.gz
0742e51bfdc91f8521c51f88d8dd33cf1359e0f989cedcfa83e42a086989cab2  guix-build-3f90ddea8a6a/output/arm-linux-gnueabihf/SHA256SUMS.part
f6520f99a71bd81099f7ca057b7afedda8c04f8667655ccf85261a71d5b50098  guix-build-3f90ddea8a6a/output/arm-linux-gnueabihf/bitcoin-3f90ddea8a6a-arm-linux-gnueabihf-debug.tar.gz
e1f059327ff8c0d37517f147bdb3658269b32a589a6269b9714a86b597eaf901  guix-build-3f90ddea8a6a/output/arm-linux-gnueabihf/bitcoin-3f90ddea8a6a-arm-linux-gnueabihf.tar.gz
691e71ee82756c0da6ce088845d7381f754f54e643c5a3dc5e651dfde1cd4ffa  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/SHA256SUMS.part
b461e0e99feda343ae5c4d38294ba0d6436271c1d987db2c9fe7ee522989a946  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/bitcoin-3f90ddea8a6a-arm64-apple-darwin-unsigned.dmg
6194c67f4969a20e8ed8a270e6b518daa5bf7117bf8a0dc52f5bcc3b1d17e4d5  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/bitcoin-3f90ddea8a6a-arm64-apple-darwin-unsigned.tar.gz
813d71f1a7a9305879b05287c3d1688befdf1cacd3c75efe0fe8356dc0d84ddb  guix-build-3f90ddea8a6a/output/arm64-apple-darwin/bitcoin-3f90ddea8a6a-arm64-apple-darwin.tar.gz
17125568ee8d8430064b9d907dbd8b00c27b2ec51279312d88a5485ca4cc50e5  guix-build-3f90ddea8a6a/output/dist-archive/bitcoin-3f90ddea8a6a.tar.gz
0b312c9b82982fb28ac0c4b0a2956ca41fec4dcfab920b63b7e5253735452120  guix-build-3f90ddea8a6a/output/powerpc64-linux-gnu/SHA256SUMS.part
efa52ae550949b9b335a64811f38e031fbc2669714401ed26b995ffd604602be  guix-build-3f90ddea8a6a/output/powerpc64-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64-linux-gnu-debug.tar.gz
54739498f4bf1f3b733e597c57334e984fe8033929390eb4704a18ef0f152ff3  guix-build-3f90ddea8a6a/output/powerpc64-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64-linux-gnu.tar.gz
621f0e34fdca59c9446b0768fc0ff8175ed9179d3a5f85bedc4492f72adddeed  guix-build-3f90ddea8a6a/output/powerpc64le-linux-gnu/SHA256SUMS.part
0f374f8c5d6796b93aef42f0da5e8cee6fa44fe71198c1c0eb90163da7d7f024  guix-build-3f90ddea8a6a/output/powerpc64le-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64le-linux-gnu-debug.tar.gz
cc6881853aec9c39945194b439fe28ab327bcbf65907fedb9861166f4101001a  guix-build-3f90ddea8a6a/output/powerpc64le-linux-gnu/bitcoin-3f90ddea8a6a-powerpc64le-linux-gnu.tar.gz
93bb658a02282960f8110d0f14abac379693c8830cefa2de72e0d4ff1bc35d5c  guix-build-3f90ddea8a6a/output/riscv64-linux-gnu/SHA256SUMS.part
8dd848e5e57dd42e5f9960efd8e9e3b12540bfcd6bbd358462dea4fac5af2a8d  guix-build-3f90ddea8a6a/output/riscv64-linux-gnu/bitcoin-3f90ddea8a6a-riscv64-linux-gnu-debug.tar.gz
0180d91f039bd27ed201ca86b939f70bb0eabd023ffa264ada4276682c308dd3  guix-build-3f90ddea8a6a/output/riscv64-linux-gnu/bitcoin-3f90ddea8a6a-riscv64-linux-gnu.tar.gz
fba6ac013dc8a60752f418825fc6d45745918045c85408fb171b8c3dca13c86a  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/SHA256SUMS.part
9fd387d226cb0d4b5d7cfaae3b7de67e67d737026146d4f63234abd242e05655  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/bitcoin-3f90ddea8a6a-x86_64-apple-darwin-unsigned.dmg
8c1e6444a7e696d5ee96c6c4349d541c935660206c0cb4beb67a1393e22e9dbc  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/bitcoin-3f90ddea8a6a-x86_64-apple-darwin-unsigned.tar.gz
b579bbff39bfa905be1e6c6cecf1df40ddbb2c18337ff569183f5a3cec15d17f  guix-build-3f90ddea8a6a/output/x86_64-apple-darwin/bitcoin-3f90ddea8a6a-x86_64-apple-darwin.tar.gz
4085d46573c68d99a91341b0887e4e9613e39d6a2d9889f029adbf92dc87900c  guix-build-3f90ddea8a6a/output/x86_64-linux-gnu/SHA256SUMS.part
ac8683a938f33ab14da2ff62cb96e5655817aa755ef6d649db2b320a1d7b0f43  guix-build-3f90ddea8a6a/output/x86_64-linux-gnu/bitcoin-3f90ddea8a6a-x86_64-linux-gnu-debug.tar.gz
43ca0d57ae2a29a2b7ba4985f3bfa48a0acb9c9cf75e90bff7f4042d6990bfac  guix-build-3f90ddea8a6a/output/x86_64-linux-gnu/bitcoin-3f90ddea8a6a-x86_64-linux-gnu.tar.gz
60ed831c18881a3f552ab7a8bfffddd3f6e6c3095fbd17e95995f2819a885876  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/SHA256SUMS.part
59b19c36e73f7e00a507307580242752685f4e17fa21e3f6851b82c1fc095aea  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64-debug.zip
2ef548bee7f9f7420fa5336c3d830c6d40147d5082346ab8b594d38ef0ede74a  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64-setup-unsigned.exe
c4bc61a85609e838bc82c0fd9b73c43dbcc73261cb43bffda7ac6c2681de0eb0  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64-unsigned.tar.gz
0ae766295c82818aba22e2a5ed139a739fd1cfb5d0293d808408d9b46432c62d  guix-build-3f90ddea8a6a/output/x86_64-w64-mingw32/bitcoin-3f90ddea8a6a-win64.zip
```

achow101 commented:
Are you still working on this?

hebasto commented:
> Are you still working on this?

I am. This PR fixes a bug in our depends build system, and it's ready for further reviewing.

fanquake commented on depends/packages/qt.mk:
```diff
@@ -263,7 +263,6 @@ endef
 
 define $(package)_config_cmds
   export PKG_CONFIG_SYSROOT_DIR=/ && \
```
While we are here, should we also move `PKG_CONFIG_SYSROOT_DIR` into _config_env?

hebasto committed:
build: Make $(package)_*_env available to all $(package)_*_cmds

hebasto committed:
build: Move environment variables into `$(package)_config_env`

hebasto commented:
Updated 3f90ddea8a6a2061cfb347a1d77df2c0a6fa238c -> affbf58a1e52a8e60c830be6a9e0347e0ff4c28e ([pr24279.04](https://github.com/hebasto/bitcoin/commits/pr24279.04) -> [pr24279.05](https://github.com/hebasto/bitcoin/commits/pr24279.05)):

- addressed @fanquake's [comment](https://github.com/bitcoin/bitcoin/pull/24279#discussion_r1042337073)

hebasto commented:
Guix builds:
```
0c15350386dcec6e7a9fb4640c481fdd765571eee3da35ff148899c51555cb46  guix-build-affbf58a1e52/output/aarch64-linux-gnu/SHA256SUMS.part
b152c462526637d0fb9dc46e0c273de843a2abbb02fae4f480e5bd77d8de59bd  guix-build-affbf58a1e52/output/aarch64-linux-gnu/bitcoin-affbf58a1e52-aarch64-linux-gnu-debug.tar.gz
f14c25104a3b51bc82cb108683292e64d6c27060acab930412c25425a1a3bd1a  guix-build-affbf58a1e52/output/aarch64-linux-gnu/bitcoin-affbf58a1e52-aarch64-linux-gnu.tar.gz
53c66796a2c2f7bad4c8d5f278c84b415e96d63f9e6cbad4236b453cf4d01c2f  guix-build-affbf58a1e52/output/arm-linux-gnueabihf/SHA256SUMS.part
84fd91ebfcaa5b87ea56c510e0d0cb058987713973b0d27fc5aca5350f605f9a  guix-build-affbf58a1e52/output/arm-linux-gnueabihf/bitcoin-affbf58a1e52-arm-linux-gnueabihf-debug.tar.gz
c343fd593204132a0d7e3df8a6550d3b58912c21f56ab14635cdc27644d780e4  guix-build-affbf58a1e52/output/arm-linux-gnueabihf/bitcoin-affbf58a1e52-arm-linux-gnueabihf.tar.gz
e6470535a01054e783177f7b5ac8873e69a8c2726174c988410ea38e9551afd9  guix-build-affbf58a1e52/output/arm64-apple-darwin/SHA256SUMS.part
adbea1d95a6e57dc6d470c760cb03e7a27cbd44227c497bdfb78382ad4f97ba5  guix-build-affbf58a1e52/output/arm64-apple-darwin/bitcoin-affbf58a1e52-arm64-apple-darwin-unsigned.dmg
fdd0c22d1f34a44234b3e41ed236d9f69ee8f1a9f26f6a158bbcfb16e823977a  guix-build-affbf58a1e52/output/arm64-apple-darwin/bitcoin-affbf58a1e52-arm64-apple-darwin-unsigned.tar.gz
7e889d308b77c19303eaf0ae7285b369d4437bf0a23ddf86cdcade66956ae54c  guix-build-affbf58a1e52/output/arm64-apple-darwin/bitcoin-affbf58a1e52-arm64-apple-darwin.tar.gz
fa8574dabfa9c866b64f2eedf23e42d70406f1e040d30c33197ebf28842139fc  guix-build-affbf58a1e52/output/dist-archive/bitcoin-affbf58a1e52.tar.gz
5a64fb39f4eac60b11591a85797230cd596d217087dfe50d41e556d2b94e99fd  guix-build-affbf58a1e52/output/powerpc64-linux-gnu/SHA256SUMS.part
5fc641b36ba2236a2dabd012d406106eef09ecdd569587bd3f5762673b321185  guix-build-affbf58a1e52/output/powerpc64-linux-gnu/bitcoin-affbf58a1e52-powerpc64-linux-gnu-debug.tar.gz
9dc88283445a3ccc16a72a85d2718aee95b190e7c1ce2e7451cc8befd233237d  guix-build-affbf58a1e52/output/powerpc64-linux-gnu/bitcoin-affbf58a1e52-powerpc64-linux-gnu.tar.gz
eb73f47ffb50e8d6027d6f0f83cec52669b4b8ea6b5ff4a73305618889183bea  guix-build-affbf58a1e52/output/powerpc64le-linux-gnu/SHA256SUMS.part
7022959bce165310b9ffda49ff98f6d098915bb14ec86aa9484045fd92b8dc7f  guix-build-affbf58a1e52/output/powerpc64le-linux-gnu/bitcoin-affbf58a1e52-powerpc64le-linux-gnu-debug.tar.gz
646c3e154f7d8d608a2ad93eaf7b4836abf28143270aa19d76068c9307578772  guix-build-affbf58a1e52/output/powerpc64le-linux-gnu/bitcoin-affbf58a1e52-powerpc64le-linux-gnu.tar.gz
0cfab9d97decac6769de24dd9e5e9dc5d4d433c13f14716484cfb5727cd23d9d  guix-build-affbf58a1e52/output/riscv64-linux-gnu/SHA256SUMS.part
d4a338f25de0cca8718b677ce95245f6e88f032c283a44bc1c46d5a892037b52  guix-build-affbf58a1e52/output/riscv64-linux-gnu/bitcoin-affbf58a1e52-riscv64-linux-gnu-debug.tar.gz
851beaf6fd881d47bd0954af8ee22e23ef3a074ca63a56709d6e8527cc01313b  guix-build-affbf58a1e52/output/riscv64-linux-gnu/bitcoin-affbf58a1e52-riscv64-linux-gnu.tar.gz
496ce8af1b9d042e0c380170818be3b0a4e8ac2131b3878be0900fb4a524047b  guix-build-affbf58a1e52/output/x86_64-apple-darwin/SHA256SUMS.part
09b3beafd3fa1d9d026290d75e29876dc9e1ece9e2e85bdfc22296e346705f6a  guix-build-affbf58a1e52/output/x86_64-apple-darwin/bitcoin-affbf58a1e52-x86_64-apple-darwin-unsigned.dmg
5f28b4eb9bdc8fa0e5fbbf1a177994253d3e82b4dc4a1cdc1001377fa7ee45b0  guix-build-affbf58a1e52/output/x86_64-apple-darwin/bitcoin-affbf58a1e52-x86_64-apple-darwin-unsigned.tar.gz
d86358a39884cc9e48434f831397191548a51a313c0a95342502584eb4382325  guix-build-affbf58a1e52/output/x86_64-apple-darwin/bitcoin-affbf58a1e52-x86_64-apple-darwin.tar.gz
6e3b1c432fb1d87e0db1a7db17111b5cd548703350bd3acb1abef965f966c546  guix-build-affbf58a1e52/output/x86_64-linux-gnu/SHA256SUMS.part
9ef848a3d5319f75d32cec20eaaedb5575c18f68a2ef4a4de687303c130f5833  guix-build-affbf58a1e52/output/x86_64-linux-gnu/bitcoin-affbf58a1e52-x86_64-linux-gnu-debug.tar.gz
e9d762a013f547fe9bd2d675c9f60fdded54784ea73285bb9cb7ac548299d929  guix-build-affbf58a1e52/output/x86_64-linux-gnu/bitcoin-affbf58a1e52-x86_64-linux-gnu.tar.gz
2ba19870c57f6f6efa02d587ff79085990c9b9824b7708c9925d6a631013405d  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/SHA256SUMS.part
62d9567069156a089d70ea802d4b44a3bf3a5ad869fe649ebea564f3a50404fd  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64-debug.zip
287bce6249347862033d4b403c7a4fc8211b1a5cd86b52e9836768a0d2cbf8c2  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64-setup-unsigned.exe
aab154e667389456866543d23b62e7d819bc0d09cf3f81b693aaa31743be5766  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64-unsigned.tar.gz
51f7c527bd661647868941fa1cc0de96dd93019720054004a2b2d537ed27ba97  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64.zip
```

fanquake commented:
Guix Build (aarch64):
```bash
0c15350386dcec6e7a9fb4640c481fdd765571eee3da35ff148899c51555cb46  guix-build-affbf58a1e52/output/aarch64-linux-gnu/SHA256SUMS.part
b152c462526637d0fb9dc46e0c273de843a2abbb02fae4f480e5bd77d8de59bd  guix-build-affbf58a1e52/output/aarch64-linux-gnu/bitcoin-affbf58a1e52-aarch64-linux-gnu-debug.tar.gz
f14c25104a3b51bc82cb108683292e64d6c27060acab930412c25425a1a3bd1a  guix-build-affbf58a1e52/output/aarch64-linux-gnu/bitcoin-affbf58a1e52-aarch64-linux-gnu.tar.gz
53c66796a2c2f7bad4c8d5f278c84b415e96d63f9e6cbad4236b453cf4d01c2f  guix-build-affbf58a1e52/output/arm-linux-gnueabihf/SHA256SUMS.part
84fd91ebfcaa5b87ea56c510e0d0cb058987713973b0d27fc5aca5350f605f9a  guix-build-affbf58a1e52/output/arm-linux-gnueabihf/bitcoin-affbf58a1e52-arm-linux-gnueabihf-debug.tar.gz
c343fd593204132a0d7e3df8a6550d3b58912c21f56ab14635cdc27644d780e4  guix-build-affbf58a1e52/output/arm-linux-gnueabihf/bitcoin-affbf58a1e52-arm-linux-gnueabihf.tar.gz
e6470535a01054e783177f7b5ac8873e69a8c2726174c988410ea38e9551afd9  guix-build-affbf58a1e52/output/arm64-apple-darwin/SHA256SUMS.part
adbea1d95a6e57dc6d470c760cb03e7a27cbd44227c497bdfb78382ad4f97ba5  guix-build-affbf58a1e52/output/arm64-apple-darwin/bitcoin-affbf58a1e52-arm64-apple-darwin-unsigned.dmg
fdd0c22d1f34a44234b3e41ed236d9f69ee8f1a9f26f6a158bbcfb16e823977a  guix-build-affbf58a1e52/output/arm64-apple-darwin/bitcoin-affbf58a1e52-arm64-apple-darwin-unsigned.tar.gz
7e889d308b77c19303eaf0ae7285b369d4437bf0a23ddf86cdcade66956ae54c  guix-build-affbf58a1e52/output/arm64-apple-darwin/bitcoin-affbf58a1e52-arm64-apple-darwin.tar.gz
fa8574dabfa9c866b64f2eedf23e42d70406f1e040d30c33197ebf28842139fc  guix-build-affbf58a1e52/output/dist-archive/bitcoin-affbf58a1e52.tar.gz
5a64fb39f4eac60b11591a85797230cd596d217087dfe50d41e556d2b94e99fd  guix-build-affbf58a1e52/output/powerpc64-linux-gnu/SHA256SUMS.part
5fc641b36ba2236a2dabd012d406106eef09ecdd569587bd3f5762673b321185  guix-build-affbf58a1e52/output/powerpc64-linux-gnu/bitcoin-affbf58a1e52-powerpc64-linux-gnu-debug.tar.gz
9dc88283445a3ccc16a72a85d2718aee95b190e7c1ce2e7451cc8befd233237d  guix-build-affbf58a1e52/output/powerpc64-linux-gnu/bitcoin-affbf58a1e52-powerpc64-linux-gnu.tar.gz
eb73f47ffb50e8d6027d6f0f83cec52669b4b8ea6b5ff4a73305618889183bea  guix-build-affbf58a1e52/output/powerpc64le-linux-gnu/SHA256SUMS.part
7022959bce165310b9ffda49ff98f6d098915bb14ec86aa9484045fd92b8dc7f  guix-build-affbf58a1e52/output/powerpc64le-linux-gnu/bitcoin-affbf58a1e52-powerpc64le-linux-gnu-debug.tar.gz
646c3e154f7d8d608a2ad93eaf7b4836abf28143270aa19d76068c9307578772  guix-build-affbf58a1e52/output/powerpc64le-linux-gnu/bitcoin-affbf58a1e52-powerpc64le-linux-gnu.tar.gz
0cfab9d97decac6769de24dd9e5e9dc5d4d433c13f14716484cfb5727cd23d9d  guix-build-affbf58a1e52/output/riscv64-linux-gnu/SHA256SUMS.part
d4a338f25de0cca8718b677ce95245f6e88f032c283a44bc1c46d5a892037b52  guix-build-affbf58a1e52/output/riscv64-linux-gnu/bitcoin-affbf58a1e52-riscv64-linux-gnu-debug.tar.gz
851beaf6fd881d47bd0954af8ee22e23ef3a074ca63a56709d6e8527cc01313b  guix-build-affbf58a1e52/output/riscv64-linux-gnu/bitcoin-affbf58a1e52-riscv64-linux-gnu.tar.gz
496ce8af1b9d042e0c380170818be3b0a4e8ac2131b3878be0900fb4a524047b  guix-build-affbf58a1e52/output/x86_64-apple-darwin/SHA256SUMS.part
09b3beafd3fa1d9d026290d75e29876dc9e1ece9e2e85bdfc22296e346705f6a  guix-build-affbf58a1e52/output/x86_64-apple-darwin/bitcoin-affbf58a1e52-x86_64-apple-darwin-unsigned.dmg
5f28b4eb9bdc8fa0e5fbbf1a177994253d3e82b4dc4a1cdc1001377fa7ee45b0  guix-build-affbf58a1e52/output/x86_64-apple-darwin/bitcoin-affbf58a1e52-x86_64-apple-darwin-unsigned.tar.gz
d86358a39884cc9e48434f831397191548a51a313c0a95342502584eb4382325  guix-build-affbf58a1e52/output/x86_64-apple-darwin/bitcoin-affbf58a1e52-x86_64-apple-darwin.tar.gz
6e3b1c432fb1d87e0db1a7db17111b5cd548703350bd3acb1abef965f966c546  guix-build-affbf58a1e52/output/x86_64-linux-gnu/SHA256SUMS.part
9ef848a3d5319f75d32cec20eaaedb5575c18f68a2ef4a4de687303c130f5833  guix-build-affbf58a1e52/output/x86_64-linux-gnu/bitcoin-affbf58a1e52-x86_64-linux-gnu-debug.tar.gz
e9d762a013f547fe9bd2d675c9f60fdded54784ea73285bb9cb7ac548299d929  guix-build-affbf58a1e52/output/x86_64-linux-gnu/bitcoin-affbf58a1e52-x86_64-linux-gnu.tar.gz
2ba19870c57f6f6efa02d587ff79085990c9b9824b7708c9925d6a631013405d  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/SHA256SUMS.part
62d9567069156a089d70ea802d4b44a3bf3a5ad869fe649ebea564f3a50404fd  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64-debug.zip
287bce6249347862033d4b403c7a4fc8211b1a5cd86b52e9836768a0d2cbf8c2  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64-setup-unsigned.exe
aab154e667389456866543d23b62e7d819bc0d09cf3f81b693aaa31743be5766  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64-unsigned.tar.gz
51f7c527bd661647868941fa1cc0de96dd93019720054004a2b2d537ed27ba97  guix-build-affbf58a1e52/output/x86_64-w64-mingw32/bitcoin-affbf58a1e52-win64.zip
```

fanquake reviewed (approved):
ACK affbf58a1e52a8e60c830be6a9e0347e0ff4c28e

Onyeali commented:
Approve

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
From: Michael Ford ***@***.***>
Sent: Thursday, December 8, 2022 5:19:10 PM
To: bitcoin/bitcoin ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [bitcoin/bitcoin] build: Make `$(package)_*_env` available to all `$(package)_*_cmds` (PR #24279)


@fanquake approved this pull request.

affbf58<https://github.com/bitcoin/bitcoin/commit/affbf58a1e52a8e60c830be6a9e0347e0ff4c28e>

—
Reply to this email directly, view it on GitHub<https://github.com/bitcoin/bitcoin/pull/24279#pullrequestreview-1210362782>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXWG52EXRXEJULR6E7CTRODWMIC75ANCNFSM5NVZ7VRQ>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

0xB10C commented on depends/packages/qt.mk:
```diff
@@ -261,9 +262,6 @@ define $(package)_preprocess_cmds
 endef
 
 define $(package)_config_cmds
-  export PKG_CONFIG_SYSROOT_DIR=/ && \
```
fwiw: moving this to depends/funcs.mk broke my Nix depends build. I'm not sure why.

https://github.com/0xB10C/bitcoind-gunix/issues/1#issuecomment-1714381895
